packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/undefined-this-computed-class-key/input.mjs

```diff
@@ -0,0 +1 @@
+export class C { [this.name] = 42 }
```

Comment:
Can you add a test case for computed class method?
```
export class C { [this.name]() {} }
```

Reply:
@JLHwung Changed the test to the one for computed class method. PTAL.